proof/invariant-abstract/AARCH64/ArchKHeap_AI.thy

```diff
@@ -808,7 +808,8 @@ lemma aobjs_of_lift_aobj_at:
   apply (simp flip: aobjs_of_ako_at_Some)
   done
 
-lemma valid_arch_state_lift_aobj_at:
+(* intended for inside Arch, this architecture does not need cap preservation for valid_arch_state *)
+lemma valid_arch_state_lift_aobj_at_no_caps:
```

Comment:
Hmm, it might be an artefact of seeing this comment as part of a PR review instead of in Isabelle, but it should possibly be changed to make it clear that it is implicitly comparing `valid_arch_state_lift_aobj_at_no_caps` to the following interface lemma. As someone just reading through things in order, without that information the comment on its own confused me.

Reply:
changed to:
```
(* intended for use inside Arch, as opposed to the interface lemma valid_arch_state_lift_aobj_at,
   since this architecture does not need cap preservation for valid_arch_state *)
```